src/components/GlobalNavbar/SignInButton.tsx

```diff
@@ -1,18 +1,9 @@
-import styled from 'styled-components';
 import useAuthModal from 'hooks/useAuthModal';
+import TextButton from 'components/core/Button/TextButton';
 
 function SignInButton() {
   const displayAuthModal = useAuthModal();
-  return <StyledButton onClick={displayAuthModal}>Sign In</StyledButton>;
+  return <TextButton onClick={displayAuthModal} text="Sign In"></TextButton>;
```

Comment:
if there isn't children content, you can make the component self-closing: `<TestButton ... />`

Reply:
good call, addressed in separate pr: https://github.com/mikeydub/gallery/pull/73